README.md

```diff
@@ -12,6 +12,11 @@ React component for [Chartist.js](https://gionkunz.github.io/chartist-js/)
 ```
 $ npm install react-chartist --save
 ```
+You need to install chartist if you do not have it installed
```

Comment:
maybe worth note "as a peer dependencie"

Reply:
I don't follow. Do you mean add it to the README "if you do not have it installed as a peer dependency"?

Reply:
I mean to point out `chartist` is a peer dependency that has to be install separately in order to use `react-chartist`.